Resizetizer.NT/SkiaSharpBitmapTools.cs

```diff
@@ -1,53 +1,35 @@
 ﻿using SkiaSharp;
 using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Text;
+using System.Diagnostics;
 
 namespace Resizetizer
 {
-	internal class SkiaSharpBitmapTools : IDisposable
+	internal class SkiaSharpBitmapTools : SkiaSharpTools, IDisposable
```

Comment:
Base type of `SkiaSharpTools` does all the work, we just load, provide the size and draw.